### PR TITLE
Create header and footer for platform pages

### PR DESCRIPTION
**Pull request by nathan-barrett**
## Description
Certain pages on getpocket.com use a separate header and footer from other pages which use a global nav. 

- [x] rearranged how we're bundling the css so we're just serving the styles for one page at a time.
- [x] create header
- [x] create footer

## Issue / Bugzilla link
#10783
## Testing
https://localhost:8000/externalpages/pocket/add

### CODE REVIEW

File: bedrock/externalpages/templates/externalpages/pocket/includes/platform-footer.html
Comment (reemhamz) [inline bedrock/externalpages/templates/externalpages/pocket/includes/platform-footer.html:21]: `..social-media--list`

File: bedrock/externalpages/templates/externalpages/pocket/includes/platform-nav.html
Comment (reemhamz) [inline bedrock/externalpages/templates/externalpages/pocket/includes/platform-nav.html:15]: `..nagivation--list`

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (reemhamz) [inline media/css/externalpages/pocket/components/_platform-footer.scss:8]: We should either create a variable holding this hexcode, or use a pre-defined one. `$color-marketing-gray-30` from Protocol looks very similar to it, so you can use that here

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (reemhamz) [inline media/css/externalpages/pocket/components/_platform-footer.scss:11]: I realize now that we should probably stick to pre-defined tokens from Protocol if we're able to. We can use `$spacing-xl` here 🪐

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (reemhamz) [inline media/css/externalpages/pocket/components/_platform-footer.scss:25]: Remember to update this to `&--links` if you end up changing the BEM convention

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (reemhamz) [inline media/css/externalpages/pocket/components/_platform-footer.scss:28]: We can replace this with `@include text-body-xs;`

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (reemhamz) [inline media/css/externalpages/pocket/components/_platform-footer.scss:47]: We can erase this extra white space

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (reemhamz) [inline media/css/externalpages/pocket/components/_platform-footer.scss:76]: Can we just use a black color here?

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (reemhamz) [inline media/css/externalpages/pocket/components/_platform-footer.scss:82]: I think instead of using `text-indent`, we should probably just create aria-labels for the icons, or use a span for the text with a class of `sr-only` to hide the text. What do you think?

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (reemhamz) [inline media/css/externalpages/pocket/components/_platform-footer.scss:72]: Instead of using a background for social icons, could we possibly just borrow social icons from something like Font Awesome? I see that you're using background sizing, positioning, etc, and that is probably an over-kill just to display the icons. I would say to just borrow the icons, and place `aria-labels` in the element holding them for better accessibility.

File: media/css/externalpages/pocket/components/_platform-nav.scss
Comment (reemhamz) [inline media/css/externalpages/pocket/components/_platform-nav.scss:10]: Same here with the hexcode as I mentioned above, let's find a nice token from Protocol to use instead! It helps with consistency :)

**Review by reemhamz — CHANGES_REQUESTED** [review]
You did a great job with displaying the content! However, let's focus on replacing the units (in pixels) and colors with tokens already available in Protocol as much as we can? I didn't comment on each one, so there's more to replace than I commented on. I realize now that we should do that as much as we can for better consistency since Protocol is already greatly integrated into Bedrock. 

Otherwise, r+wc! 💫 We're getting closer and closer : )

File: bedrock/externalpages/templates/externalpages/pocket/includes/platform-footer.html
Comment (nathan-barrett) [reply bedrock/externalpages/templates/externalpages/pocket/includes/platform-footer.html:9]: I like it!

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (nathan-barrett) [reply media/css/externalpages/pocket/components/_platform-footer.scss:82]: I might try and replace this with the @text-hidden helper from protocol, just need to set it up first. But good call!

File: media/css/externalpages/pocket/components/_platform-nav.scss
Comment (nathan-barrett) [reply media/css/externalpages/pocket/components/_platform-nav.scss:10]: Yeah - worry is changing what the hex is on the pocket site without approval on changing the color. Thought was to get it up and running and when the redesign happens we can make an actual pocket theme - since none of the colors in protocol are actually _for_ pocket.

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (nathan-barrett) [reply media/css/externalpages/pocket/components/_platform-footer.scss:76]: Let me look around and see if theres something close enough

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (nathan-barrett) [reply media/css/externalpages/pocket/components/_platform-footer.scss:8]: Same thought with changing color to a protocol hex - I'm wary to change it. But I can bring it up with Craig/Alex

Comment (nathan-barrett) [reply media/css/externalpages/pocket/components/_platform-footer.scss:8]: Ok - just talked with Craig about it. Since these codes are apart of the pocket "theme" they should be hard coded until these hexes are added to protocol.

File: media/css/externalpages/pocket/components/_platform-footer.scss
Comment (nathan-barrett) [reply media/css/externalpages/pocket/components/_platform-footer.scss:11]: Good lookin out

File: bedrock/externalpages/templates/externalpages/pocket/includes/platform-footer.html
Comment (reemhamz) [inline bedrock/externalpages/templates/externalpages/pocket/includes/platform-footer.html:22]: This should be a `span` element because it's not really considered a list item, so it doesn't make sense to put it in `li` tags
```suggestion
        <li class="platform-footer__navigation--social-media--list-text">Connect:</li>
```
```suggestion
        <span class="platform-footer__navigation--social-media--list-text">Connect:</span>
```

File: bedrock/externalpages/templates/externalpages/pocket/includes/platform-footer.html
Comment (reemhamz) [inline bedrock/externalpages/templates/externalpages/pocket/includes/platform-footer.html:30]: Semantically, this should also probably be a `span`

**Review by reemhamz — CHANGES_REQUESTED** [review]
Just a couple of more _nits_ that I should have pointed out during my first review, but everything seems good to me now and a lot cleaner! Gg! r+wc and then we'll be good to merge 🪐✨

**Pull request merged**
